codecs/packbits/README.md

```diff
@@ -0,0 +1,67 @@
+# packbits codec
+
+Defines an `array -> bytes` codec that packs together values smaller than 1
```

Comment:
This could be clearer given that this codec now supports the 12-bit `complex_float6_e2m3fn` and `complex_float6_e3m2fn` data types